parm/analysis/gsi/gsiparm.anl.tmp

```diff
@@ -17,12 +17,12 @@
   lrun_subdirs=.true.,
   netcdf_diag=_NETCDF_DIAG_,binary_diag=_BINARY_DIAG_,
   newpc4pred=.true., adp_anglebc=.true., angord=4,
-  passive_bc=.false., use_edges=.false., emiss_bc=.true.,
-  diag_precon=.true., step_start=1.e-3, upd_pred(1)=0,
-  upd_pred(2)=0,upd_pred(3)=0,upd_pred(4)=0,
-  upd_pred(5)=0,upd_pred(6)=0,upd_pred(7)=0,
-  upd_pred(8)=0,upd_pred(9)=0,upd_pred(10)=0,
-  upd_pred(11)=0,upd_pred(12)=0,
+  passive_bc=.true., use_edges=.false., emiss_bc=.true.,
+  diag_precon=.true., step_start=1.e-3, upd_pred(1)=1,
+  upd_pred(2)=1,upd_pred(3)=1,upd_pred(4)=1,
+  upd_pred(5)=1,upd_pred(6)=1,upd_pred(7)=1,
+  upd_pred(8)=1,upd_pred(9)=1,upd_pred(10)=1,
+  upd_pred(11)=1,upd_pred(12)=1,
```

Comment:
It is important to not change answers in the default version of HAFS, which currently uses the GDAS satellite bias correction coefficients. Please set the above options at the configuration file level (e.g., in parm/hafs.conf) and use the existing values (e.g., passive_bc=.false.). You could consider making a new configuration file (e.g., parm/hafs_online_bias_corr.conf) where you set different values for when online satellite bias correction is used. The user could then pass this configuration file to the scripts in their cron script if desired.

Reply:
Got it. I've made such a conf. file, but I'm wondering what "tag" I should precede these options with (like how [config] precedes online_satbias=no).

I'm thinking [gsi] for these options and [gsi_vr] for the options in gsiparm.anl.tmp. Do you happen to know?

Reply:
@jak5808 that sounds like a reasonable approach to me – thanks!